web/src/javascript/leaderboardApp.js

```diff
@@ -11,9 +11,17 @@ function displayEntries(entries) {
   }
 }
 
+function clearEntries() {
+  const leaderboard = document.getElementById("leaderboard-body");
+  while (leaderboard.firstChild) {
+    leaderboard.removeChild(leaderboard.firstChild);
+  }
```

Comment:
I'm kinda sad that this is the [recommended way](http://stackoverflow.com/a/3955238/395386)... :/